src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1046,7 +1046,7 @@ class SelectClauseSegment(ansi.SelectClauseSegment):
     """Overrides ANSI to allow INTO as a terminator."""
 
     match_grammar = StartsWith(
-        Sequence("SELECT", Ref("WildcardExpressionSegment", optional=True)),
+        "SELECT",
```

Comment:
There is no point in ending in an optional part in `StartsWith`.

Nothing really to do with this, but noticed when working on this.